MODULE.bazel

```diff
@@ -6,13 +6,13 @@ bazel_dep(name = "rules_scala_annex")
 
 bazel_dep(name = "buildifier_prebuilt", version = "8.0.1", dev_dependency = True)
 
-rules_scala_annex_version = "lucid_2024-12-06"
+rules_scala_annex_version = "upgrade-to-bazel-8"
```

Comment:
- [ ] Wait until `upgrade-to-bazel-8` is merged and change this to a release tag.